libs/ui-lib/lib/ocm/components/clusterConfiguration/CpuArchitectureDropdown.tsx

```diff
@@ -169,7 +169,8 @@ const CpuArchitectureDropdown = ({
     );
     if (
       !isSelectedCpuArchitectureSupported &&
-      selectedCpuArchitecture !== getDefaultCpuArchitecture()
+      selectedCpuArchitecture !== getDefaultCpuArchitecture() &&
```

Comment:
I'm a bit off the loop ATM, but couldn't this exception be included as part of `isCpuArchitectureSupported` ?
It would make the code more modular.

Reply:
I've changed it